pages/authorDashboard/PKPAuthorDashboardHandler.inc.php

```diff
@@ -284,6 +286,32 @@ public function setupTemplate($request)
             $canEditPublication = false;
         }
 
+        $authorListGetParams = [
+            'contextIds' => $request->getContext()->getId(),
+            'count' => 30,
+        ];
+
+        $authorItems = [];
+        foreach ($latestPublication->getData('authors') as $contributor) {
+            $authorItems[] = Repo::author()->getSchemaMap()->map($contributor);
+        }
+
+        $authorCollector = Repo::author()->getCollector();
+        $authorCollector->filterByPublicationIds([$latestPublication->getId()]);
+        $authorItemsMax = Repo::author()->getCount($authorCollector->limit(null)->offset(null));
+        $contributorsListPanel = new \PKP\components\listPanels\PKPContributorsListPanel(
+            'contributors',
+            __('publication.contributors'),
+            [
+                'form' => $contributorForm,
+                'getParams' => $authorListGetParams,
+                'items' => $authorItems,
+                'itemsMax' => $authorItemsMax,
```

Comment:
`itemsMax` is not supported by the contributors list panel.

Reply:
Removed